Misc/NEWS.d/next/Library/2022-09-25-23-24-52.gh-issue-97545.HZLSNt.rst

```diff
@@ -0,0 +1 @@
+Make Semaphore run faster.
```

Comment:
This is very vague and confusing as what Semaphore is made to run faster? `threading` module or `asyncio`?

Reply:
Sorry, that's a good point. It's asyncio.Semaphore. You should be able to create a PR that updates just that particular NEWS file. (And backport it.)